src/windows/quiccrypto/makefile.vs

```diff
@@ -31,6 +31,12 @@ libquiccrypto_code.lib: $(SOURCES:.c=.obj) $(PLATFORM_OBJS)
   
 libquiccrypto.dll: libquiccrypto_code.lib libquiccrypto.def dllmain.obj
   link /nologo /dll /debug:full /out:libquiccrypto.dll libquiccrypto_code.lib dllmain.obj /def:libquiccrypto.def /OPT:ICF /OPT:REF ntdll.lib
+
+test.exe: test.obj libquiccrypto.dll
+  link /nologo /ltcg /debug:full /out:test.exe test.obj libquiccrypto.lib
+
+test: test.exe libquiccrypto.dll
+  test.exe
```

Comment:
I would prefix with `./` for non-windows systems

Reply:
No need - this makefile.vs is only used by nmake, which is a Win32 app.  The whole directory only builds using nmake inside a Visual Studio build environment.

Reply:
oh, right. looks good to me then!